avm/src/daemon/mod.rs

```diff
@@ -1,3 +1,5 @@
+#[cfg_attr(linux, path = "linux.rs")]
+#[cfg_attr(not(linux), path = "rest.rs")]
```

Comment:
"rest" is a really weird name here. I'd immediately jump to REST and think it's related to the control port that will soon exist. :)